### ISSUE TITLE
`rule-name`: `prefer-minimal-ternary` is a bit too aggressive by default

### ISSUE THREAD

**Issue by alfaproject** [sindresorhus/eslint-plugin-unicorn#3372]
### Description

I'm happy for the new option on the latest release but I was hoping we could also have an option to leave functions alone like in this example from the docs:
```
// ❌
const foo = test ? a() : b();

// ✅
const foo = (test ? a : b)();
```

I don't find the new version any more readable and in fact it breaks some of the grepping patterns I use, like if I'm searching for `a(` that won't work anymore after the suggested change. I'm happy that search for references still work if you use TypeScript, but code search using grep-like tools is much faster for me in most cases.

Perhaps this could be another flag?

### Examples

See description above

**Comment by Zamiell** [comment]
I agree in that the `(test ? a : b)()` recommendation from the README is difficult to read!
However, personally, I think that this example should be written like this:

```ts
const func = test ? a : b; 
func();
```

This way, it better satisfies the Single Line Responsibility principal: https://midu.dev/single-line-responsability-haz-una-cosa-por-linea/

I'll admit that in this trivial example, breaking this out seems kind of stupid. With that said, in most cases I think you would be passing arguments to the function. So splitting it up can really help the readability! For example:

```ts
const getBeverage = user.likesBeer ? getBeer : getWine; 
getBeverage(someArg1, someArg2, someArg3);
```
